src/sentry/integrations/vsts/integration.py

```diff
@@ -304,30 +304,61 @@ def build_integration(self, state):
             ).exists()
 
         except (IntegrationModel.DoesNotExist, AssertionError):
-            subscription_id, subscription_secret = self.create_subscription(
-                instance, account['AccountId'], oauth_data)
+            shared_secret = create_subscription_secret()
```

Comment:
If we intend for these secrets to be per-subscription, should we be creating two here? One for the WorkItem subscription and one for the PR subscription?